src/common/partitionKeys.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * The default path for the partition key if the user does not specify one on container create.
+ * @hidden
+ * @internal
+ */
+export const DEFAULT_PARTITION_KEY_PATH = "/_partitionKey";
+/**
+ * Header value used for indicating undefined partition key value
+ * @hidden
+ * @internal
+ */
+export const EMPTY_PARTITION_KEY = [{}] as [{}];
```

Comment:
EMPTY is known as `[]` and UNDEFINED is known as `[{}]` by the internal team

The name this is given in all the other SDKs is `NONE_PARTITION_KEY_VALUE`, but I'd imagine we should just handle undefined instead and not have a well known constant we use.

Reply:
Removed entirely